tools/measurement/apksize/apksize.gradle

```diff
@@ -33,6 +33,9 @@ apply from: "src/functions/functions.gradle"
  * @param -Ppull_request the pull request number to be included in the report
  */
 task generateMeasurements(type: GenerateMeasurementsTask) {
+    description 'Builds all supported variants and calculates APK sizes.'
+    group 'ApkSize'
```

Comment:
I think we should use a single group for all metrics. Either Metrics or Measurements sounds good to me.

Reply:
Done. They are all within group "Measurements" now.

Note: I have to rename the tasks as `{generate, upload}{ApkSize, Coverage}Measurements` since tasks with duplicate names under same category do not show properly with `./gradlew tasks`. This also means we need to update the prow config accordingly.